api/api.py

```diff
@@ -68,6 +69,25 @@ class SortBy(str, Enum):
     ASCENDING_DATE = "ascending_date"
 
 
+class CreateSuggestionBody(BaseModel):
+    title: str = Field(
+        None, title="Suggestion title", description="Suggestion title",
+    )
+    email_address: str = Field(
+        None, title="Email address", description="Email address who is sending email"
+    )
+    name: str = Field(
+        None, title="Name", description="Name who is sending email",
+    )
+    content: str = Field(
+        None, title="Email content", description="Email content with suggestion",
+    )
```

Comment:
Noticed that missing fields are allowed right now, if I understood correctly, setting the default values forces them to be required, right? I think this should be the case, what do you think?